addons/project/static/src/views/project_task_list/project_task_list_renderer.js

```diff
@@ -14,6 +14,12 @@ export class ProjectTaskListRenderer extends ListRenderer {
                 (task) => getRawValue(task, "project_id") !== projectId
             );
         }
+        if (column.name === "sale_line_id" && selection.length) {
```

Comment:
This should be in the sale_project module.

cc @MissingNoShiny

Reply:
changes have been made

cc @MissingNoShiny